app/helpers/automated_tests_helper.rb

```diff
@@ -48,12 +48,11 @@ def update_test_groups_from_specs(assignment, test_specs)
         display_output = extra_data_specs['display_output'] || TestGroup.display_outputs.keys.first
         test_group_name = extra_data_specs['name'] || TestGroup.model_name.human
         criterion_id = nil
-        criterion_type = nil
         if !extra_data_specs['criterion'].nil? && extra_data_specs['criterion'].include?('_')
-          criterion_id, criterion_type = extra_data_specs['criterion'].split('_') # polymorphic field
```

Comment:
@david-yz-liu How do I test whether this breaks anything? There aren't any tests that currently call this function (directly or indirectly in the controller methods)

Reply:
This has to do with autotesting, so don't worry about testing this one right now.